files/ko/web/api/baseaudiocontext/index.html

```diff
@@ -0,0 +1,122 @@
+---
+title: BaseAudioContext
+slug: Web/API/BaseAudioContext
+tags:
+  - API
+  - Audio
+  - BaseAudioContext
+  - Context
+  - Interface
+  - Reference
+  - Web Audio API
+  - sound
+browser-compat: api.BaseAudioContext
+---
+<div>{{APIRef("Web Audio API")}}</div>
+
+<p class="summary"><span class="seoSummary"><a href="/ko/docs/Web/API/Web_Audio_API">Web Audio API</a>의 <code>BaseAudioContext</code> 인터페이스는, 각각 {{domxref("AudioContext")}} 와 {{domxref("OfflineAudioContext")}}에 의해 표현되는, 온라인과 오프라인 오디오 프로세싱 그래프에 대한 기본 정의로 작동합니다. <code>BaseAudioContext</code>는 직접적으로 사용될 수 없습니다 — 이 두 가지 상속되는 인터페이스 중 하나를 통해 이것의 기능들을 사용할 수 있습니다.</p>
```

Comment:
```suggestion
<p class="summary"><span class="seoSummary"><a href="/ko/docs/Web/API/Web_Audio_API">Web Audio API</a>의 <code>BaseAudioContext</code> 인터페이스는 {{domxref("AudioContext")}} 와 {{domxref("OfflineAudioContext")}}에 의해 표현되는 온라인과 오프라인 오디오 프로세싱 그래프에 대한 기본 정의로 작동합니다.  <code>BaseAudioContext</code>는 직접적으로 사용될 수 없습니다. 대신 위에서 언급한 두 상속 인터페이스를 통해  <code>BaseAudioContext</code>의 기능을 사용할 수 있습니다.</p>
```
쉼표 (`,`)가 문장을 읽을때 더욱 이해하기 어렵게 만들고 있습니다. 해당 내용을 제거해도 충분히 원문의 의미를 남길 수 있을 것 같습니다.
두번째 문장에서는 번역체 느낌이 나 이를 자연스럽게 수정했습니다.